bindata/network/ovn-kubernetes/common/001-crd.yaml

```diff
@@ -3150,4 +3150,384 @@ spec:
       storage: true
       subresources:
         status: {}
-{{- end }}
\ No newline at end of file
+{{- end }}
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
I think we want to enable CEL validaiton waiting for k8s 1.31 upstream, as it is already available in openshift

Reply:
Done

CRD definition is updated accordingly